fr.dwightstudio.JArmEmu.json

```diff
@@ -0,0 +1,70 @@
+{
+  "id" : "fr.dwightstudio.JArmEmu",
+  "runtime" : "org.freedesktop.Platform",
+  "runtime-version": "23.08",
+  "sdk" : "org.freedesktop.Sdk",
+  "command": "jarmemu",
+  "sdk-extensions" : [ "org.freedesktop.Sdk.Extension.openjdk" ],
+  "finish-args": [
+    "--env=PATH=/app/jre/bin:/app/bin:/usr/bin",
+    "--env=JAVA_HOME=/app/jre",
+    "--env=JAVA=/app/jre/bin/java",
+    "--share=ipc",
+    "--socket=x11",
+    "--socket=pulseaudio",
+    "--share=network",
+    "--device=dri",
+    "--filesystem=home",
+    "--persist=.java"
+  ],
+  "modules": [
+    {
+      "name" : "openjdk",
+      "buildsystem" : "simple",
+      "build-commands" : [ "/usr/lib/sdk/openjdk/install.sh" ]
+    },
+    {
+      "name": "flatpak-xdg-utils",
+      "buildsystem": "meson",
+      "sources": [
+        {
+          "type": "git",
+          "url": "https://github.com/flatpak/flatpak-xdg-utils.git",
+          "tag": "1.0.5"
+        }
+      ]
+    },
+    {
+      "name": "jarmemu",
+      "buildsystem": "simple",
+      "build-commands": [
+        "/usr/lib/sdk/openjdk/install.sh",
+        "mkdir -p /app/share/java/JArmEmu/",
+        "cp -r java/JArmEmu /app/share/java/",
+        "cp -r icons/ /app/share/",
+        "cp -r mime/ /app/share/",
+        "install jarmemu /app/bin/jarmemu",
+        "patch --verbose /app/bin/jarmemu script.patch"
+      ],
+      "post-install": [
+        "install -Dm644 fr.dwightstudio.JArmEmu.metainfo.xml /app/share/metainfo/fr.dwightstudio.JArmEmu.metainfo.xml",
+        "desktop-file-install --dir=/app/share/applications fr.dwightstudio.JArmEmu.desktop"
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://github.com/Dwight-Studio/JArmEmu/releases/download/v0.1.4/JArmEmu-0.1.4-BETA.flatpak-source.zip",
```

Comment:
since this seems to be "specific" to flatpak (according to the name) why doens't it include the metainfo.xml (which is not flatpak specific) nor an already fixed launch script.